owasp-top10-2017-apps/a7/comment-killer/README.md

```diff
@@ -0,0 +1,58 @@
+# Comment-killer
+
+Comment-killer is a simple JS app that contains an example of multiple Cross-Site Scripting vulnerabilities and its main goal is to describe how a malicious user could exploit them on this purposefully vulnerable app.
+
+<img src="image/img1.png" alt="img1.png"/>
+
+## What is Cross-Site Scripting?
+
+XSS flaws occur whenever an application includes untrusted data in a new web page without proper validation or escaping, or updates an existing web page with user-supplied data using a browser API that can create HTML or JavaScript. XSS allows attackers to execute scripts in the victim’s browser which can hijack user sessions, deface web sites, or redirect the user to malicious sites.
+
+The main goal of this app is to discuss how **Cross-Site Scripting** vulnerabilities can be exploited and to encourage developers to send secDevLabs Pull Requests on how they would mitigate these flaws. Learn more <a href="https://owasp.org/www-community/attacks/xss/">here</a>.
+
+## Setup
+
+To start this intentionally **insecure application**, you will need [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/). After forking [secDevLabs](https://github.com/globocom/secDevLabs), you must type the following commands to start:
+
+`cd secDevLabs/owasp-top10-2017-apps/a7/comment-killer `
+
+`make install `
+
+Then simply visit [http://localhost:3000/](http://localhost:3000/) ! 😆
```

Comment:
Could you please change the app's port to `10007`, since we changed it in the `Docker-Compose` file?